### ISSUE TITLE
Mouse scroll wheel now scrolls horizontally by default

### ISSUE THREAD

**Issue by nosklo** [ragardner/tksheet#32]
In version <= 4.9.1, scrolling the mouse wheel scrolls the sheet vertically by default, as expected, similar to excel.

But in version 4.9.2 the mouse wheel is now scrolling horizontally by default

I can confirm the issue still exists in the latest version (4.9.4 right now) - it can be reproduced by running the default demo then scrolling your mouse wheel.

**Comment by nosklo** [comment]
The issue still exists even if I disable the horizontall scrollbar (with `show_x_scrollbar=False` argument)

**Comment by ragardner** [comment]
On my machine (windows 10) it only scrolls horizontally with the shift key held down, I can add an option to disable horizontal scrolling if required?

**Comment by nosklo** [comment]
I'm also using windows 10, and it scrolls horizontally by default here. Even with shift pressed, it still scrolls horizontally.
I can work around the issue by monkeypatching the function with the older version:

from tksheet import MainTable

```py
def mousewheel(self, event=None):
    if event.num == 5 or event.delta == -120:
        self.yview_scroll(1, "units")
        self.RI.yview_scroll(1, "units")
    if event.num == 4 or event.delta == 120:
        if self.canvasy(0) <= 0:
            return
        self.yview_scroll(-1, "units")
        self.RI.yview_scroll(-1, "units")
    self.main_table_redraw_grid_and_text(redraw_row_index=True)


MainTable.mousewheel = mousewheel
```

by adding this before creating my sheet, scrolling works fine.

**Comment by ragardner** [comment]
That is odd... you may have to use your patch until I can figure out what's going on, I don't want to break the change I made for someone else who wanted touch pad and Mac OS scrolling

**Comment by PatLechevallier** [comment]
Hi @nosklo Can you tell us the value of event.state when you scroll with the mouse or horizontally (mouse+shift pressed) ? 
And as well as the brand of the machine you use, that could help @ragardner

**Comment by ragardner** [comment]
@nosklo @PatLechevallier 

I have attempted to fix the issues in `4.9.5` by specifically binding shift-mousewheel

**Comment by nosklo** [comment]
Sorry for the delay on the verification.

This 4.9.5 update completely fixes the issue for me. Now horizontal scroll works by default.
